src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Server/Data/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace BlazorWasm_CSharp.Server.Data
+{
+    public class WeatherForecast
```

Comment:
This class isn't used either, right? We have one in the 'shared' project.

Reply:
It is on the controller, unless I'm confused

Reply:
You're right. I've removed it.